lib/traject/dor_config.rb

```diff
@@ -20,26 +20,18 @@
   next if resource.collection?
```

Comment:
I think @jkeck said the `display_type` was effectively unused; should we remove it?